speech-api/SpeechSynthesis-speak-without-activation-fails.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
```

Comment:
As discussed in https://github.com/w3c/speech-api/pull/28#issuecomment-410660492, maybe make this test tenatative (name it "SpeechSynthesis-speak-without-activation-fails.tentative.html") and link to an open issue in the speech-api repo.

Reply:
Done.